blog-posts/thoughs-on-collaborative-work.md

```diff
@@ -0,0 +1,81 @@
+---
+path: "/blog/thoughts-on-collaborative-work/"
+date: "2022-02-24"
+title: "Thoughts on collaborative work"
+author: Jesús Real
+authorSummary: Senior Frontend Engineer @ Satellytes
+featuredImage: images/thoughs-on-collaborative-work/group-table.jpg
+attribution:
+    creator: Marvin Meyer
+    source: https://unsplash.com/photos/SYTO3xs06fU
+seoMetaText: This article aims to offer an overview about the common beliefs around collaborative work, highlight the (many) (surprising) positive sides of it, and depicts some on-field examples that will either suit you or inspire you to find your own ways on approaching collaborative work. 
+teaserText: This article aims to offer an overview about the common beliefs around collaborative work, highlight the (many) (surprising) positive sides of it, and depicts some on-field examples that will either suit you or inspire you to find your own ways on approaching collaborative work.
+leadboxText: Are you excited about getting the most out your team work? Would you like to work with great professionals? Then join us!
+
+
+---
+
+Most of us developers, despite being more or less aware of the benefits of doing more work together, work on our own most of the time. Why? What can we do about it? Let's dive into it!
+
+## Is it just about programming?
+
+Short answer, no, it’s not. As software developers we don’t just write code, we also play a crucial role in actively shaping, challenging and clarifying requirements. And this happens in many different flavours: emails, issue tracking systems, chat tools, team meetings, inter-team meetings... Every area of our work can benefit if we approach it in a more collaborative way.
+
+## The good, the bad and the ugly. Or is it just good?
```

Comment:
I love the western but I don't get the headline in those aspects. Maybe less abstract?

Reply:
I like it, I would like to keep it as it is.

Reply:
I like the new introduction revolving around impostor but I must say I still don't get the headline. What are the three aspects you refer to ? Do you mean the bullet list included?

Reply:
I have commented outside of the review already. Please help here. I guess I'm not the only one with problems to understand this headline? Maybe ask colleagues? I might be wrong here.

Reply:
"The good, the bad and the ugly" is perfectly fine. It's a very well known movie title and user all over the place, over and over again. Approval from my side.

Reply:
FYI in the meantime I updated it to "The good, the bad and the ugly about collaboration", instead of "The good, the bad and the ugly. Or is it just good?"